src/LinuxCaptureService/MemoryWatchdog.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef LINUX_CAPTURE_SERVICE_MEMORY_WATCHDOG_H_
+#define LINUX_CAPTURE_SERVICE_MEMORY_WATCHDOG_H_
+
+#include <optional>
+#include <string_view>
+#include <thread>
```

Comment:
We should also include stdint.h here. And thread is probably not needed.

Reply:
Done.